### ISSUE TITLE
Key transform resets type in fast_jsonapi 1.3

### ISSUE THREAD

**Issue by hult** [Netflix/fast_jsonapi#310]
In fast_jsonapi 1.3 (but not in 1.2), calling `set_key_transform` after `set_type` seems to reset the type

```
class DeckSerializer
  include FastJsonapi::ObjectSerializer
  set_type :decks
  set_key_transform :dash
end

DeckSerializer.new(Deck.first).serializable_hash
=> {:data=>{:id=>1, :type=>:deck}}
```

Note the singular "deck". I expected the plural "decks".

If I switch the order of `set_type` and `set_key_transform`, it works.

**Comment by hult** [comment]
Sorry, been travelling. Now I see a fix is underway, thanks! :)

**Comment by shishirmk** [comment]
Released in [Version 1.4](https://github.com/Netflix/fast_jsonapi/releases/tag/1.4)

**Comment by hult** [comment]
It seems the fix didn't make version 1.4 after all?
